klustair/templates/frontend-deployment.yaml

```diff
@@ -83,12 +83,12 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           securityContext:
             capabilities:
-              add: ["CHOWN", "SETGID", "SETUID", "NET_BIND_SERVICE"]
               drop: ["ALL"]
             allowPrivilegeEscalation: false
             privileged: false
             readOnlyRootFilesystem: false
-            runAsNonRoot: false
+            runAsUser: 101
```

Comment:
Is there a way to configure if the frontend is deliverd by nginx or apache?
I did not find a setting but based on it, one could dynamically put the correct uid in here